cmd/samproxy/main.go

```diff
@@ -37,6 +37,7 @@ type Options struct {
 }
 
 func main() {
+	fmt.Println("Starting samproxy")
```

Comment:
should this be using a logger?

Reply:
There's no logger available there. I was using it for testing to make sure _something_ spun up; I don't think it's necessary anymore.  Removing it.